_posts/blog/2018-09-18/2018-09-18-5 -tips-to-make-your-openbazaar-listings-pop.markdown

```diff
@@ -0,0 +1,158 @@
+---
+title: "<span class='post-title-extra'>5 Tips</span> To Make Your OpenBazaar Listings <span class='post-title-extra'>Pop</span>"
+layout: post
+date: '2018-09-18 06:30:00 -0300'
+social_title: '5 Tips To Make Your OpenBazaar Listings POP!'
+social_description: 'With OpenBazaar, the largest decentralized marketplace in the world, you can build your store and listings faster than on any other network and we have some tips that will help your store stand out.'
+social_image: OpenBazaar Store on OpenBazaar.png
+---
+
+_By Monique Boediono, OpenBazaar UX Designer_
+
+Ecommerce is an exciting and flexible way to start your own business. With some ecommerce business models you can even earn money from the comfort of your own home. While there are many opportunities to get started though, there are also many ways to get lost in the crowd. 
+
+![OpenBazaar Store on OpenBazaar](OpenBazaar Store on OpenBazaar.png "OpenBazaar Store on OpenBazaar")
+
+With OpenBazaar, the world's largest decentralized marketplace, you can build your store and listings faster than on any other network and we have some tips that will help your store stand out.
+
+If you have something you’ve been itching to sell on OpenBazaar, here are 5 tips to help make your listings **POP** on the network! 🎈
+
+![Creating a Listing Title on OpenBazaar](Creating Listing Tags on OpenBazaar.gif "Creating a Listing Title on OpenBazaar")
+
+### 1. Write a descriptive title
+
+Your title should give your buyers a clear idea of what you’re selling. When writing a title, try throwing in a few descriptive keywords to give your listing some extra oomph. For example, “hand-crafted pure copper jug with lid” is a lot more interesting and helpful than just “water jug”.
+
+### 2. Take many, many photos 📸
+
+If a picture is worth a thousand words and you can attach up to 30 photos for every listing, that should save you 30,000 words, right? (And way too many headaches.) Take advantage of this.
```

Comment:
The () is part of the sentence, and is the second half of "should save you". This should be "...30,000 words (and way too many headaches), right?"